docs/index.md

```diff
@@ -22,18 +16,21 @@ Install the Kili client with pip:
 pip install kili
 ```
 
-## Usage
+## Getting Started
 
-Instantiate the Kili client:
+- Create and copy a [Kili API key](https://docs.kili-technology.com/docs/creating-an-api-key)
+- Add the `KILI_API_KEY` variable in your bash environment (or in the settings of your favorite IDE) by pasting the API key value you copied above:
 
-```python
-from kili.client import Kili
-kili = Kili()
-# You can now use the Kili client!
-```
+  ```bash
+  export KILI_API_KEY='<you api key value here>'
+  ```
```

Comment:
Why removing the doc about instantiating Kili without the API key ?

Reply:
indeed, I'll change that